<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_m">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>模拟</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tw_show">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>631</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_1">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>22</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>城市1</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>城市2</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_plane">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>飞机</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>航班</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_user">
       <property name="text">
        <string>用户</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ticket">
       <property name="text">
        <string>订单</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="l_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>城市3</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_4">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>194</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>城市4</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>城市5</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>城市6</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>航班号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_pn">
       <property name="text">
        <string> </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>btn_m</zorder>
   <zorder>tw_show</zorder>
   <zorder>l_1</zorder>
   <zorder>l_2</zorder>
   <zorder>l_3</zorder>
   <zorder>l_4</zorder>
   <zorder>l_5</zorder>
   <zorder>l_6</zorder>
   <zorder>l_plane</zorder>
   <zorder>label</zorder>
   <zorder>l_pn</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1159</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
